_posts/2020-05-27-ols1-wrapup.md

```diff
@@ -0,0 +1,46 @@
+Through this program, OLS helped design and implement projects to solve technical and cultural questions by collaboratively designing open source software, AI techniques, educational resources, culture movements, and global partnerships in their communities (detail: https://openlifesci.org/ols-1/projects-participants/). 
+
+The final step to graduate Open Life Science is a short presentation about what you worked on. While we originally envisioned this being in April, COVID-19 brought a whole new set of pressures, so we decided to extend the deadlines for final presentations, as well as offering additional timeslots. The result - five calls - can all be viewed on [our YouTube channel](https://www.youtube.com/playlist?list=PL1CvC6Ez54KB6U9GtjOjwESMurHgT41qM&advanced_settings=1&disable_polymer=1)
+ 
+Some of the common themes that surprised us and brought us joy during the presentations: 
+- The COVID-19 pandemic had an unexpected impact of our program beyond the initial goals as our mentees could transfer their OLS skills in other areas like establishing effective communication among the international COVID collaborations, designing the format of their lab meetings online, setting mentor-mentee relationships in their local networks, and connecting folk musicians and artists with each other in Nepal during the lockdown.
+- The [open canvas](https://docs.google.com/presentation/d/1MeJo0TyuMg_waLk1J4q9y1aAqKNMuRBlnmxEChSz-cQ/edit#slide=id.p) exercise was commonly useful for everyone. We presented this in [our first cohort call](https://openlifesci.org/ols-1/week02/), but a large number of mentees found it so foundational they shared it even 3-4 months later in their presentations. 
+- Skills like collaborative work culture, self care, ally skills, giving and receiving feedback, agile management, and knowledge dissemination are valuable beyond research.  
+A Youtube Stream can make all the difference in helping your friends and family attend important events: 
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">The graduation was even more special because my <a href="https://twitter.com/hashtag/VUamsterdam?src=hash&amp;ref_src=twsrc%5Etfw">#VUamsterdam</a> family, <a href="https://twitter.com/gravana?ref_src=twsrc%5Etfw">@gravana</a>, and <a href="https://twitter.com/PhDToothFAIRy?ref_src=twsrc%5Etfw">@PhDToothFAIRy</a><br>were there 😍<br>AND, my mother watched my graduation in real-time. She had to miss both my wedding and PhD defense. It&#39;s the first event in my life abroad that she witnessed 👩‍👧 <a href="https://t.co/xWWaJ9kLhw">pic.twitter.com/xWWaJ9kLhw</a></p>&mdash; Lena Karvovskaya (@LangData) <a href="https://twitter.com/LangData/status/1263468708330078209?ref_src=twsrc%5Etfw">May 21, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> 
```

Comment:
I could change a bit the CSS to center it. What do you think?

Reply:
Sounds great!